x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/metrics.tsx

```diff
@@ -102,33 +100,50 @@ const TabComponent = (props: TabProps) => {
     [setTime]
   );
 
+  const timeRange = {
+    interval: '1m',
+    to: currentTime,
+    from: currentTime - time,
+    ignoreLookback: true,
+  };
+
+  const defaultMetrics: Array<{ type: SnapshotMetricType }> = [
+    { type: 'rx' },
+    { type: 'tx' },
+    buildCustomMetric('system.cpu.user.pct', 'user'),
+    buildCustomMetric('system.cpu.system.pct', 'system'),
+    buildCustomMetric('system.load.1', 'load1m'),
+    buildCustomMetric('system.load.5', 'load5m'),
+    buildCustomMetric('system.load.15', 'load15m'),
+    buildCustomMetric('system.memory.actual.used.bytes', 'usedMemory'),
+    buildCustomMetric('system.memory.actual.free', 'freeMemory'),
+    buildCustomMetric('system.cpu.cores', 'cores', 'max'),
+  ];
+
   const { nodes, reload } = useSnapshot(
     filter,
-    [
-      { type: 'rx' },
-      { type: 'tx' },
-      buildCustomMetric('system.cpu.user.pct', 'user'),
-      buildCustomMetric('system.cpu.system.pct', 'system'),
-      buildCustomMetric('system.load.1', 'load1m'),
-      buildCustomMetric('system.load.5', 'load5m'),
-      buildCustomMetric('system.load.15', 'load15m'),
-      buildCustomMetric('system.memory.actual.used.bytes', 'usedMemory'),
-      buildCustomMetric('system.memory.actual.free', 'freeMemory'),
-      buildCustomMetric('system.cpu.cores', 'cores', 'max'),
-    ],
+    [...defaultMetrics, ...customMetrics],
     [],
     nodeType,
     sourceId,
     currentTime,
     accountId,
     region,
     false,
-    {
-      interval: '1m',
-      to: currentTime,
-      from: currentTime - time,
-      ignoreLookback: true,
-    }
+    timeRange
+  );
+
+  const { nodes: logRateNodes, reload: reloadLogRate } = useSnapshot(
+    filter,
+    [{ type: 'logRate' }],
+    [],
+    nodeType,
+    sourceId,
+    currentTime,
+    accountId,
+    region,
+    false,
+    timeRange
```

Comment:
Just out of curiosity, why do we need to do a separate request for log rate?

Reply:
LogRate is basically counting all the documents in the index for the specified time range. This is problematic when logRate is calculated with other metrics (like cpu and memory in the same request) and it will include not just the count for "logs" but also the count for "metric" events; it uses both metricAlias and logAlias index patterns from the source configuration. https://github.com/elastic/kibana/issues/80304#issuecomment-797009916

Reply:
Thanks a lot for the explanation! 🙌